docs/undergrad.yaml

```diff
@@ -219,7 +219,7 @@ Bilinear and Quadratic Forms Over a Vector Space:
     norm: 'inner_product_space.of_core.to_has_norm'
     orthonormal bases: 'maximal_orthonormal_iff_basis_of_finite_dimensional'
   Endomorphisms:
-    orthogonal group: ''
+    orthogonal group: 'linear_isometry_equiv.group'
```

Comment:
Let's leave this off til we have the isomorphism between this and `unitary.group` (which should be very quick, just no one's done it)